02-express-tutorial/app.js

```diff
@@ -1 +1,50 @@
-console.log('Express Tutorial')
+const { products } = require("./data");
+const express = require('express');
+const app = express();
+
+app.listen(3000, () => {
+    console.log('server is on');
+});
+app.use(express.static("./public"));
+app.get('/api/v1/test', (req, res)=>{
+    res.json({ message: "It worked!" });
+});
+app.get('/api/v1/products', (req, res)=>{
+    res.json({products });
```

Comment:
I don't think you should need the brackets here around products, but if you tested and it worked not a blocker!

Reply:
it worked, but if it was not correct I will fixed it. Thank you!)